freedomofpress/securedrop
# i18n: add flags for a few more languages

## Status

Ready for review

## Description of Changes

Slightly adapted from:

https://www.sketchappsources.com/free-source/823-flag-set-sketch-freebie-resource.html

## Testing

* vagrant ssh development
* cd /vagrant/securedrop
* add **SUPPORTED_LOCALES = ['es_ES', 'nb_NO', 'nl', 'ar', 'fr_FR', 'zh_Hant', 'en_US']** in config.py
* ./manage.py run
* firefox http://localhost:8080
* verify that all locales have a flag when clicking on the menu

![a](https://user-images.githubusercontent.com/433594/33126434-c3c42f92-cf84-11e7-89d9-9ce7534be777.png)

## Deployment

None

## Checklist

### If you made changes to the app code:
